cypress/e2e/case-list/persistSearchPanel.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { ResolutionStatus } from "types/ResolutionStatus"
+import { defaultSetup, loginAndGoToUrl } from "../../support/helpers"
+
+const caseTriggers: { code: string; status: ResolutionStatus }[][] = [
+  [
+    {
+      code: "TRPR0001",
+      status: "Unresolved"
+    }
+  ]
+]
+
+before(() => {
+  defaultSetup()
+})
+
+beforeEach(() => {
+  cy.task("clearCourtCases")
+})
+
+it("Should show search panel by default after logging in", () => {
+  loginAndGoToUrl()
+
+  cy.contains("Apply filters")
+})
+
+it("Should persist the closure of the side panel when the user refreshes the page", () => {
```

Comment:
This test should probably assert that the side panel isn't visible as well

Reply:
done